lib/phoenix/presence.ex

```diff
@@ -79,6 +79,18 @@ defmodule Phoenix.Presence do
 
   See `c:list/1` for more information on the presence data structure.
 
+  ## Custom dispatcher
+
+  It's possible to customize the dispatcher module used to broadcast.
+  By default, `Phoenix.Channel.Server` is used, which is the same dispatcher
+  used by channels. To customize the dispatcher, pass the `:dispatcher` option
+  when using `Phoenix.Presence`:
+
+      use Phoenix.Presence,
+        otp_app: :my_app,
+        pubsub_server: MyApp.PubSub,
+        dispatcher: MyApp.CustomDispatcher
```

Comment:
We need some documentation or links on what is the dispatch and how a custom implementation would look like. I don't know if we want to link to the endpoint, Elixir's Registry or what else, but we just need a bit more context here. :)